generators/example/src/main/scala/Configs.scala

```diff
@@ -247,3 +247,12 @@ class RV32BoomAndRocketConfig extends Config(
   new freechips.rocketchip.subsystem.WithRV32 ++
   new freechips.rocketchip.subsystem.WithNBigCores(1) ++
   new freechips.rocketchip.system.BaseConfig)
+
+class RocketL2Config extends Config(
```

Comment:
Can these be put in the BOOM, Rocket and (BOOM and Rocket) areas in the config file respectively? I think we should avoid having alot of configs spread out in a bunch of places.

Reply:
Putting all rocket-chip configs in BOOM seems like its going to cause trouble eventually.

Reply:
Sorry if this was not clear. I mean just put these new configs in the sections related to boom/rocket/etc in this example config file.

Reply:
I pushed a fix.